modules/launcher/src/main/java/org/mule/runtime/module/launcher/service/FileSystemServiceProviderDiscoverer.java

```diff
@@ -56,6 +58,11 @@ public FileSystemServiceProviderDiscoverer(ArtifactClassLoader containerClassLoa
 
     final List<ServiceDescriptor> serviceDescriptors = new LinkedList<>();
 
+    File servicesFolder = getServicesFolder();
```

Comment:
Not sure that this class must be on launcher module. Looks like the only reason is because of this dependency on MuleFoldersUtil.